File: solutions/credentials/ruby/credentials.rb
module Lab
  class Credentials

    attr_reader :client_id
    attr_reader :client_secret
    attr_reader :token_server_url
    attr_reader :livestream_endpoint_url

    def initialize
      @client_id = "356e592973-summitlab"
      @client_secret = "2f0bfb2dba04bf30d714"
      @token_server_url = "https://api.omniture.com/token"
      @livestream_endpoint_url = "https://sjo.livestream.adobe.net/api/1/stream/dt-lab-prime"
    end

  end
end

File: credentials/java/lab/Credentials.java
package lab;

public class Credentials {

  protected String clientId;
  protected String clientSecret;
  protected String tokenServerUrl;
  protected String livestreamEndpointUrl;

  public Credentials() {
    this.clientId = "356e592973-summitlab";
    this.clientSecret = "2f0bfb2dba04bf30d714";
    this.tokenServerUrl = "https://api.omniture.com/token";
    this.livestreamEndpointUrl = "https://sjo.livestream.adobe.net/api/1/stream/dt-lab-prime";
  }

  public String getClientId() { return this.clientId; }
  public String getClientSecret() { return this.clientSecret; }
  public String getTokenServerUrl() { return this.tokenServerUrl; }
  public String getLivestreamEndpointUrl() { return this.livestreamEndpointUrl; }
}
